drivers/ukintctlr/gic/gic-v2.c

```diff
@@ -43,15 +43,18 @@
 #include <uk/plat/common/acpi.h>
 #endif /* CONFIG_UKPLAT_ACPI */
 #include <uk/plat/common/bootinfo.h>
-#include <uk/plat/common/irq.h>
 #include <uk/plat/spinlock.h>
 #include <arm/cpu.h>
+#include <uk/intctlr.h>
 #include <uk/intctlr/gic-v2.h>
+#include <uk/intctlr/limits.h>
 #include <uk/ofw/fdt.h>
 
 /* Max CPU interface for GICv2 */
 #define GIC_MAX_CPUIF		8
 
+#define GIC_MAX_IRQ	UK_INTCTLR_MAX_IRQ
```

Comment:
This looks misaligned with the above macro (tabs wise).

Reply:
Done